<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArduinoData</class>
 <widget class="QWidget" name="ArduinoData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ArduinoData</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tWPlot">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="tabSettings">
        <attribute name="title">
         <string>Einstellungen</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Verbindung</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Geräteport</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QComboBox" name="cBDevice">
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Baudrate</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QComboBox" name="cBBaudRate">
                <property name="editable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="gBExport">
           <property name="title">
            <string>Ex&amp;port</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Dateiname</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Preferred</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="lEFileName"/>
              </item>
              <item>
               <widget class="QPushButton" name="pBFile">
                <property name="text">
                 <string>Durchsuchen</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Log</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTextEdit" name="tELog"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pBClearLog">
               <property name="text">
                <string>Leeren</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Kopieren</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabPlain">
        <attribute name="title">
         <string>Klartext</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="tEPlainData">
           <property name="plainText">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tPlot">
        <attribute name="title">
         <string>Plot</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="Plot" name="graphicsView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pBClose">
         <property name="text">
          <string>Schließen</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pBStartStop">
         <property name="text">
          <string>S&amp;tarten</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plot</class>
   <extends>QGraphicsView</extends>
   <header location="global">plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pBClose</sender>
   <signal>clicked()</signal>
   <receiver>ArduinoData</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBStartStop</sender>
   <signal>clicked()</signal>
   <receiver>ArduinoData</receiver>
   <slot>startStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cBBaudRate</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ArduinoData</receiver>
   <slot>updateBaudRate(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBClearLog</sender>
   <signal>clicked()</signal>
   <receiver>ArduinoData</receiver>
   <slot>clearLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBFile</sender>
   <signal>clicked()</signal>
   <receiver>ArduinoData</receiver>
   <slot>filePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startStop()</slot>
  <slot>updateBaudRate(QString)</slot>
  <slot>clearLog()</slot>
  <slot>filePath()</slot>
 </slots>
</ui>
